core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/Resource.scala

```diff
@@ -7,19 +7,19 @@ import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.Paths
 import java.util.zip.ZipInputStream
-
 import scala.util.control.NonFatal
-
 import pureconfig.ConfigReader
-
 import org.virtuslab.ideprobe.ConfigFormat
 import org.virtuslab.ideprobe.Extensions._
 
+import scala.annotation.nowarn
+
 sealed trait Resource
 
 object Resource extends ConfigFormat {
   implicit val resourceConfigReader: ConfigReader[Resource] = ConfigReader[String].map(from)
 
+  @nowarn // as match is not exhaustive - case class Jar is not handled
```

Comment:
Hmmm actually... maybe it should be handled here? 🤔 or if we know from somewhere else that Jar will never be passed here (doubt), then a separate trait should be extracted? pls open an issue (as I suppose it's not trivial enough to be fixed on this PR)

Reply:
Issue created: https://github.com/VirtusLab/ide-probe/issues/242 - until the issue is done, I suggest to have this `@nowarn` annotation added.